roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2

```diff
@@ -0,0 +1,181 @@
+        "prodaccepted",
+        "testaccepted"
+      ]
+    },
+    "user": {
+      "type": "string"
+    },
+    "created": {
+      "type": "string"
+    },
+    "revisionnote": {
+      "type": "string"
+    },
+    "active": {
+      "type": "boolean"
+    },
+    "notes": {
+      "type": [
+        "string",
+        "null"
+      ]
+    },
+    "metaDataFields": {
+      "type": "object",
+      "properties": {
+        "NameIDFormat": {
+          "$ref": "#/definitions/NameIDFormat"
+        },
+        "logo:0:url": {
+          "type": "string",
+          "format": "url",
+          "info": "Enter the URL to the logo used for this service. e.g. https://static.example-logo.nl/media/sp/logo.png."
+        },
+        "logo:0:width": {
+          "type": "number",
+          "info": "The width of the logo found at logo:0:url in pixels."
+        },
+        "logo:0:height": {
+          "type": "number",
+          "info": "The height of the logo found at logo:0:url in pixels."
+        },
+        "coin:institution_guid": {
+          "type": "string",
+          "format": "uuid",
+          "info": "This is a 128 bit number also known as a globally unique identifier (GUID or UUID) for this service."
+        },
+        "coin:service_team_id": {
+          "type": "string",
+          "info": "Enter the team name as used in the SP Dashboard. e.g. urn:collab:group:example-teams.nl:nl:surfnet:services:spd_example. "
+        },
+        "secret": {
+          "type": "string",
+          "format": "password",
+          "info": "The secret of this Resource Server for authentication purposes."
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "fetch": "scopes"
+          },
+          "info": "The allowed scopes for this Resource Server."
+        }
+      },
+      "patternProperties": {
+        "^name:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "Set the name of the service. Format: 'Service Name | Supplier'"
+        },
+        "^description:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The description of the service."
+        },
+        "^OrganizationName:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The formal name of the organization. e.g. Service by University of Harderwijk."
+        },
+        "^OrganizationDisplayName:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The friendly name of the organization. e.g. University of Harderwijk."
+        },
+        "^NameIDFormats:([0-2]{1})$": {
+          "$ref": "#/definitions/NameIDFormat",
+          "multiplicity": 3,
+          "info": "The NameIDFormat(s) this service provider supports. This can be transient, persistent or unspecified."
+        },
```

Comment:
Do we need the NameIDFormats in addition to the NameIDFormat?

Reply:
No, we don't. NameIDFormat is sufficient.